doc/scripts/genrest.py

```diff
@@ -135,6 +143,22 @@ def init():
 
         modules.append((title, suffix, abspath, desc_path))
 
+    no_index_modules = []
+    for module_spec in args.no_index_modules:
+        spec_parts = module_spec.split(",")
+        if len(spec_parts) == 2:
+            title, path_s = spec_parts
+        else:
+            sys.exit(f"error: --no-index-modules argument '{module_spec}' "
+                     "should have the format <title>,<path>.")
+
+        abspath = pathlib.Path(path_s).resolve()
+        if not abspath.exists():
```

Comment:
Would an `abspath.is_dir()` check be a bit better?

Reply:
Done.